content/articles/2019-09-tidyr-1-0-0.Rmarkdown

```diff
@@ -35,18 +35,21 @@ As you might guess from the version number, this is a major release, and the 1.0
 
 This blog post summarises the four major changes to the package:
 
-* New `pivot_longer()` and `pivot_wider()` supercede `spread()` and `gather()`. 
-  The new functions are substantially more powerful, thanks to ideas from the 
-  [data.table](https://CRAN.R-project.org/package=data.table) and [cdata](https://CRAN.R-project.org/package=cdata) packages, and I'm confident that you'll find them easier 
-  to use and remember than their predecessors. 
+* New `pivot_longer()` and `pivot_wider()` provide improved tools for reshaping,
+  superceding `spread()` and `gather()`. The new functions are substantially 
+  more powerful, thanks to ideas from the 
+  [data.table](https://CRAN.R-project.org/package=data.table) and
+  [cdata](https://CRAN.R-project.org/package=cdata) packages, and I'm 
+  confident that you'll find them easier to use and remember than their 
+  predecessors. 
   
-* New functions for **rectangling**, converting deeply nested lists into tidy
-  data frames: `unnest_auto()`, `unnest_longer()`, `unnest_wider()`, and
-  `hoist()`.
+* New `unnest_auto()`, `unnest_longer()`, `unnest_wider()`, and `hoist()` 
+  provide new tools for rectangling, converting deeply nested lists into tidy
+  data frames.
   
-* Interface changes to `nest()` and `unnest()` to match an emerging consensus
-  for interface design across the tidyverse. I've also discovered that nesting
-  can be broken down into two simpler steps, called `pack()` and `chop()`.
+* `nest()` and `unnest()` have been changed to match an emerging principle for 
+  the design of  `...` interfaces. Four new functions (`pack()`/`unpack()`, and
+  `chop()`/`unchop()`) reveal two simpler steps that underlying nesting.
```

Comment:
"underlie"?

Of course that look weird to me now. But it's definitely not "that underlying nesting".